yt_dlp/YoutubeDL.py

```diff
@@ -3691,23 +3677,27 @@ def run_all_pps(self, key, info, *, additional_pps=None):
         return info
 
     def pre_process(self, ie_info, key='pre_process', files_to_move=None):
+        if files_to_move is not None:
+            self.report_warning('[pre_process] "files_to_move" is deprecated and may be removed in a future version')
```

Comment:
For viz: wanted to make sure this messaging was on-brand for deprecating a _parameter_ rather than an entire method